docs/installation.rst

```diff
@@ -6,8 +6,8 @@ Installation
 
 #. Check out a clone of this repo to a location of your choice, such as
    ``git clone --depth=1 https://github.com/Bash-it/bash-it.git ~/.bash_it``
-#. Run ``~/.bash_it/install.sh`` (it automatically backs up your ``~/.bash_profile`` or ``~/.bashrc``\ , depending on your OS)
-#. Edit your modified config (\ ``~/.bash_profile`` or ``~/.bashrc``\ ) file in order to customize Bash-it.
+#. Run ``~/.bash_it/install.sh`` (it automatically backs up your ``~/.bashrc`` or ``~/.bashrc``\ , depending on your OS)
```

Comment:
Seems like you have replaced `bash_profile` string in the the files with `bashrc`. Please fix this

Reply:
Good catch! Fixed 👍